.gitignore

```diff
@@ -1,3 +1,15 @@
 .project
 .vscode
 *.env
+/Finished
+/Package
+/atlcon
+/vagrant
+/atlassian
```

Comment:
Those characters at the end look weird?

Reply:
I have removed them. This have been small computer icons. I don't know how they got there. Please approve again.